src/backend/templates/event.hbs

```diff
@@ -57,6 +57,7 @@
            <div class="col-md-4">
            </div>
            <div style="padding-right:30px" class="col-md-8">
+             <a id="ticket-button" style="display: block ; position: relative ; top: -87px ; visibility: hidden; "></a>  
              <a href="{{{eventurls.register}}}" class="pull-right btn btn-lg btn-primary" role="button">Tickets</a>
```

Comment:
extra anchor tag is needed to enable the smooth scrolling ( like the scrolling that occurs when you click the back to top link ) on the main page.